EvidenceApi/V1/Gateways/DocumentsApiGateway.cs

```diff
@@ -90,6 +90,7 @@ public async Task<List<Claim>> GetClaimsByIdsThrottled(List<string> claimIds)
                     throw new DocumentsApiException($"Incorrect status code returned: {response.StatusCode}");
                 }
 
+                await Task.Delay(200);
```

Comment:
This will work great for all the claims until the last bunch where it will add an unnecessary 200ms delay after they have finished even though there are no more requests to send. I think it might be worth thinking about whether we can just reduce the size of the `SemaphoreSlim` further and do without a delay. Also, I think a delay will introduce bursts of calls as opposed to spreading the calls more evenly.

Reply:
Lowered the number of requests from 10 to 5 and also removed the delay